funding/manager_test.go

```diff
@@ -2765,6 +2765,7 @@ func TestFundingManagerCustomChannelParameters(t *testing.T) {
 	const minHtlcIn = 1234
 	const maxValueInFlight = 50000
 	const fundingAmt = 5000000
+	const chanReserve = 100000
```

Comment:
Should we also add a test case for using a reserve that's smaller than the default 1% but above dust? So basically the absolute minimum possible?